vllm/compilation/decorators.py

```diff
@@ -302,7 +303,49 @@ def patched_inline_call(parent, func, args, kwargs):
             with patch.object(InliningInstructionTranslator, 'inline_call',
                               patched_inline_call), torch._dynamo.config.patch(
                                   **dynamo_config_patches):
+                compilation_config = self.vllm_config.compilation_config
```

Comment:
I like this location better! Maybe extract this into a context manager, it'll make the code clearer and cleaner